blocks/footer/footer.css

```diff
@@ -1,14 +1,92 @@
 footer {
-  padding: 2rem;
-  background-color: var(--overlay-background-color);
+  background-color: #3e40a0;
   font-size: var(--body-font-size-s);
+  color: white;
 }
 
-footer .footer {
+.footer {
   max-width: 1200px;
   margin: auto;
 }
 
-footer .footer p {
-  margin: 0;
-}
\ No newline at end of file
+
+.footer ul {
+  list-style-type: none;
+}
+
+.footer a {
+  color: white;
+  text-decoration: none;
+}
+
+/* main footer section that contains the different subsections */
+footer > div > div {
+  display: flex;
+  flex-flow: row wrap;
+  margin: 16px;
```

Comment:
Inner spacings should be padding. the margin overlaps and creates a white line at the bottom

Reply:
Fixed, thanks!